tests/baselines/reference/objectTypeWithCallSignatureHidingMembersOfFunctionAssignmentCompat.errors.txt

```diff
@@ -15,17 +15,19 @@ tests/cases/conformance/types/members/objectTypeWithCallSignatureHidingMembersOf
     var f: Object;
     f = i;
     i = f;
-    ~
+        ~
 !!! error TS2322: Type 'Object' is not assignable to type 'I'.
 !!! error TS2322:   The 'Object' type is assignable to very few other types. Did you mean to use the 'any' type instead?
 !!! error TS2322:     Type 'Object' provides no match for the signature '(): void'.
+!!! related TS6230 tests/cases/conformance/types/members/objectTypeWithCallSignatureHidingMembersOfFunctionAssignmentCompat.ts:8:5: Did you mean to access the property 'toString' of Object instead?

```

Comment:
Hmm, this feels not useful. I'm not sure what would be a better heuristic.

Reply:
I am working on removing suggestions that come from `Object` as `Function`. Also I think providing a list of all possible children might be a good idea, with the caveat that if there are more than 5 possible properties we don't provide this at all. I mean if there are more than 5 we are not really providing a very targeted suggestion also it probably means the target type is weak enough we probably don't know what you really meant.

Reply:
This is no longer suggested.